src/routes/lobby/LobbyView.vue

```diff
@@ -1,4 +1,23 @@
 <template>
+  <BaseDialog 
+    v-model="errorLoadingDialogVisible"
+    title="Error Joining Lobby"
+  >
+    <template #body>
+      <span>Oops. </span><br>
+      <span>Could not join that game at this time. Go home to find another one</span>
```

Comment:
The text here could use a little work, additionally spans with a break tag really aren't a great way to handle it. I would check other standard usage in the codebase and prefer titles and paragraph tags with the proper vuetify classing.